src/utils/feature-flags.ts

```diff
@@ -296,6 +296,18 @@ export const FF_LSDV_4998 = 'fflag_fix_front_lsdv_4998_missed_dynamic_children_0
  */
 export const FF_PROD_309 = 'fflag_feat_front_prod_309_choice_hint_080523_short';
 
+/**
+ * Allow to load Taxonomy from remote API
+ * @link https://app.launchdarkly.com/default/production/features/fflag_feat_front_lsdv_5451_async_taxonomy_110823_short
+ */
+export const FF_TAXONOMY_ASYNC = 'fflag_feat_front_lsdv_5451_async_taxonomy_110823_short';
+
+/**
+ * Allow to label NER directly with Taxonomy instead of Labels
+ * @link https://app.launchdarkly.com/default/production/features/fflag_feat_front_lsdv_5452_taxonomy_labeling_110823_short
+ */
+export const FF_TAXONOMY_LABELING = 'fflag_feat_front_lsdv_5452_taxonomy_labeling_110823_short';
```

Comment:
if we're updating the FF variable naming conventions can we get that updated in the FF documentation? i think this direction is better as the FF variable is a more descriptive

Reply:
100% agree, this is the better way to go. We shouldn't ever hope to have FF's around long enough to recall what it is by the ticket ids 😆

Reply:
It was described in my proposal for improved FFs, so if you like it, then that's more thumbs up for it =)